app/views/listings/show.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1= @listing.pet_name
+%p=@listing.pet_picture
+%p Dates: #{@listing.start_date} to #{@listing.end_date}
+
+%h2 Offers
+- if @listing.offers.any?
+  - @listing.offers.each do |offer|
+    .offer{id: dom_id(offer)}
```

Comment:
Are the elements below supposed to be within the `.offer?  div? Check the indentation

Reply:
Yes, they are supposed to be nested - and with current indentation they are not.. Changing the indentation now.